docs/release_notes.rst

```diff
@@ -16,6 +16,26 @@ Bug Fixes
 ^^^^^^^^^
 * Include more retiring generators in the net generation and fuel consumption
   allocation. Thanks to :user:`grgmiller` for this contirbution :pr:`3690`.
+* Added preliminary data validation checks for several FERC 1 tables that were
+  missing it:
+
+  * out_ferc1__yearly_balance_sheet_assets_sched110
```

Comment:
this is a complete personal preference but I am not sure what a user reading our release notes will get out of this list of tables. which is to say i would keep the note but not the list in here. but 1000% not a hill i will die on its so fine.